about.html

```diff
@@ -12,62 +13,109 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+  <div class="container about-page">
+
+    <div class="topnav">
+
+      <a href="./contact.html">Contact</a>
+      <a href="#">Blog</a>
+      <a href="#">Products</a>
+      <a href='./about.html'>About</a>
+      <a href="./index.html">Home</a>
+
+      <img src='./img/lambda-black.png'>
+
+    </div>
+
+    <div class="topimg">
+      <img src='./img/jumbo-about.png'>
+
+      <h2>About</h2>
+
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to
+        corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the
+        holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
 
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    </div>
 
-    Learn More
+    <div class="parent">
```

Comment:
I really like how you called the classes "parent" and "child." That makes it really easy to understand. This div could be a section tag to make the HTML more semantic